tests/cross_functional/ui/fusion_access/test_create_filesystem.py

```
@@ -38,17 +40,71 @@ class TestFDFSANConnection(ManageTest):
+        logger.info(f"Creating/updating secret '{secret_name}' in namespace '{ns}'")
+        secret_data_b64 = base64.b64encode(docker_config.encode()).decode()
+        secret_manifest = {
+            "apiVersion": "v1",
+            "kind": "Secret",
+            "metadata": {"name": secret_name, "namespace": ns},
+            "type": "kubernetes.io/dockerconfigjson",
+            "data": {".dockerconfigjson": secret_data_b64},
+        }
+        secret_yaml = json.dumps(secret_manifest)
+        ocp_obj = ocp.OCP(kind="Secret", namespace=ns)
+        ocp_obj.exec_oc_cmd(
```

Ruff (0.16.1)

[error] Function call with `shell=True` parameter identified, security issue

(S604)